src/init/features/dataconnect/sdk.ts

```diff
@@ -74,9 +71,9 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
     targetPlatform = cwdPlatformGuess;
   } else {
     // If we aren't, ask the user where their app is, and try to autodetect from there
-    logBullet(`Couldn't automatically detect your app directory.`);
+    logBullet(`Couldn't automatically detect your app in directory ${appDir}.`);
     appDir =
-      process.env[FDC_APP_FOLDER] ??
+      process.env[FDC_APP_FOLDER] ||
```

Comment:
BUG: When `process.env[FDC_APP_FOLDER]` is an empty string, `??` would take it.
BUG: in the terminal started from gen SDK button, `FDC_APP_FOLDER` is set, so subsequent init sdk call won't prompt. Logging the appDir makes it clear what's happening.